components/TodaysCard/index.js

```diff
@@ -1,59 +1,74 @@
 import Image from "next/image";
-import Link from "next/link";
-import styled from "styled-components";
-import { StyledLink, StyledListItem } from "./TodaysCard.styled";
+
+import {
+  StyledLink,
+  StyledListItem,
+  TodaysCardEventTitlePTag,
+  TodaysCardImpactPTag,
+  TodaysCardInfoContainer,
+  TodaysCardUpperContainer,
+} from "./TodaysCard.styled";
 
 export default function TodaysCard({ event, onToggleAlarm, settings }) {
+  console.log(event);
   return (
     <>
```

Comment:
I don't think we need the `React Fragment` here is everything is contained in the `StyledListItem`.